### ISSUE TITLE
Subcollections (collection groups) do not perform initial sync (?)

### ISSUE THREAD

**Issue by go-sean-go** [algolia/firestore-algolia-search#167]
I have sync'd two Firestore collections via the Algolia Firebase Extension: a standard collection (e.g. `posts`), and a subcollection (e.g. `users/{userId}/myFeed`}. Both syncs used the same Algolia App ID and scoped API Key.

Both sync properly when NEW documents are created, but only the `posts` (standard) collection did its initial sync. The `Full Index existing documents?` option was set to `true` in both cases.

Here's are the results of the subcollection:
<img width="627" alt="image" src="https://github.com/algolia/firestore-algolia-search/assets/55253273/bd0fbaae-9ab8-40d9-9065-bd9654de7333">

And here's the configuration page:
<img width="630" alt="Screenshot 2023-06-01 at 4 21 27 PM" src="https://github.com/algolia/firestore-algolia-search/assets/55253273/9703baf1-d067-4b31-b1a6-057ae5af7f99">

Again, the subcollections documents are syncing - it's just the initial sync that did nothing.

<hr> 

One other question that MAY be related: the subcollection that did not perform its initial sync MAY have documents with the same objectId - you can imagine this from the schema I posted above, wherein two users may have docs like this: `users/userA/myFeed/doc1` and `users/userB/myFeed/doc1` (aka one post can end up in many users' feeds, but it appears that all will be dumped into the same Algolia index).

I bring this up only because it's the only thing I can think of that seems like it may not align with the Algolia docs/syntax. If this is a separate matter, let me know and I'll post a separate Issue/question related to it!

**Comment by smomin** [comment]
I have fixed the defect and push it for Firebase review.  Once its reviewed, it will be available.

**Comment by guplem** [comment]
> I have fixed the defect and push it for Firebase review. Once its reviewed, it will be available.

After this patch is deployed, is there any way to force a resync? Or should we uninstall the extension and add it back again?

**Comment by go-sean-go** [comment]
@smomin Can you clarify which version (number) this will be fixed in? I see v1.1.2 is live, but I assume from the commit history we're waiting on v1.1.3 - but just want to be sure. :)

**Comment by smomin** [comment]
Yes, the fix is in v1.1.3.Sent from my iPhoneOn Jun 4, 2023, at 8:06 AM, Sean Conrad ***@***.***> wrote:﻿
@smomin Can you clarify which version (number) this will be fixed in? I see v1.1.2 is live, but I assume from the commit history we're waiting on v1.1.3 - but just want to be sure. :)
